### ISSUE TITLE
toggle bookmark doesnot actually require a payload

### ISSUE THREAD

**Issue by arush0311** [mozilla/network-pulse-api#151]
`PUT /api/pulse/entries/<id=number>/bookmark` doesnot require a payload as stated in README.md

**Comment by Pomax** [comment]
good catch: doc updates are just as important as code updates, would you be willing to correct the README.md for that?

**Comment by arush0311** [comment]
Scratch that! csrfmiddlewaretoken is required in payload. its just that when hit with rest-framework's form the csrftoken was sent in headers.. nonce on the other hand is not used..

**Comment by Avedis777** [comment]
I'm a student at Seneca college learning open source, and I was hoping to work on this bug for my course.  If no one else is currently working on it, I'd like to give it a try.

**Comment by Pomax** [comment]
Hi @Avedis777, we'd be happy to have you help out on this!

**Comment by Avedis777** [comment]
Thanks for this opportunity, i'll start working on this as soon as I can, I'm really excited.
